diff --git a/source/core.py b/source/core.py
index 9b376e5..d867944 100644
--- a/source/core.py
+++ b/source/core.py
@@ -21,7 +21,7 @@ warnings.filterwarnings("ignore")
 # - re-enable CPU fan speed display and make more generic and not only for thinkpad
 # - replace get system/CPU load from: psutil.getloadavg() | available in 5.6.2)
 
-SCRIPTS_DIR = Path("/usr/local/share/auto-cpufreq/scripts/")
+SCRIPTS_DIR = Path("/usr/libexec/auto-cpufreq/scripts/")
 
 # from the highest performance to the lowest
 ALL_GOVERNORS = ("performance", "ondemand", "conservative", "schedutil", "userspace", "powersave")
